test/c_api.cpp

```diff
@@ -35,8 +36,10 @@ void CheckResult(OgaResult* result) {
   throw std::runtime_error(OgaResultGetError(result));
 }
 
-void Test_GreedySearch_Gpt_Fp32_C_API() {
-  std::cout << "Test_GreedySearch_Gpt fp32 C API" << std::flush;
+TEST(CAPITests, GreedySearchGptFp32CAPI) {
+  std::unique_ptr<OrtEnv> g_ort_env;
+  Ort::InitApi();
```

Comment:
This function should not be called multiple times in a single process. Therefore it's better to put it in main.